### PR TITLE
switch from new app to old app commit

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/core/services/workspace/workspace.service.ts [hunk 1]
```diff
@@ -154,4 +154,12 @@ export class WorkspaceService {
     return this.storageService.get('currency');
   }
 
+  patchWorkspace(): Observable<Workspace> {
+    return this.apiService.patch(`/workspaces/${this.getWorkspaceId()}/`, {app_version: 'v1'});
+  }
+
+  getWorkspaceCreatedAt(): Date {
+    return new Date(this.storageService.get('workspaceCreatedAt'));
```
Comment (ashwin1111) [inline src/app/core/services/workspace/workspace.service.ts:162]: where are we setting value for this?

File: src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.html [hunk 2]
```diff
@@ -15,9 +15,7 @@
       </div>
       <div class="landing--xero-connect-section" (click)="connectXero()">
         <div fxLayout="row" fxLayoutAlign="center center" class="landing--xero-connect-button pointer">
-          <h3 class="landing--xero-connect-text bold single-line-text">
-            Connect to Xero Account
-          </h3>
+          <img src="../../../../assets/images/svgs/actions/connect-blue.svg">
```
Comment (ashwin1111) [inline src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.html:18]: is there any hover state available in Xero zip?

File: src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.scss [hunk 3]
```diff
@@ -50,12 +50,7 @@
     height: 48px;
   }
 
-  &--xero-connect-button {
-    background: #108000;
-    border-radius: 5px;
-    width: 318px;
-    height: 48px;
-
+  &--xero-connect-button img{
```
Comment (ashwin1111) [inline src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.scss:53]: img <space>

File: src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.html (hunk 2 above)
Comment (DhaaraniCIT) [reply src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.html:18]: no

File: src/app/core/services/workspace/workspace.service.ts (hunk 1 above)
Comment (DhaaraniCIT) [reply src/app/core/services/workspace/workspace.service.ts:162]: it is used in header component to check weather the workspace is created already or not to show the button to switch

Comment (ashwin1111) [reply src/app/core/services/workspace/workspace.service.ts:162]: okay

File: src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.scss (hunk 3 above)
Comment (ashwin1111) [reply src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.scss:53]: img <space>

Comment (ashwin1111) [reply src/app/integration/onboarding/onboarding-landing/onboarding-landing.component.scss:53]: img space {

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**